sympy/printing/latex.py

```diff
@@ -397,7 +401,14 @@ def convert(expr):
                 if self.order not in ('old', 'none'):
                     args = expr.as_ordered_factors()
                 else:
-                    args = expr.args
+                    args = list(expr.args)
+
+                # If angular quantity is present append them at the back
+                for i in range(len(args)):
+                    if isinstance(args[i], Quantity):
+                        if args[i].name.name in angular_quantity:
```

Comment:
I was thinking that printing the units at the end can be applied to all units. But if you think it's out of scope of this PR, will merge this PR.

Reply:
I got your point now . I will add other quantities in this PR only.